packages/react-accordion/src/components/Accordion/renderAccordion.tsx

```diff
@@ -2,16 +2,21 @@ import * as React from 'react';
 import { DescendantProvider, getSlots } from '@fluentui/react-utilities';
 import { AccordionState, AccordionSlots } from './Accordion.types';
 import { accordionShorthandProps } from './useAccordion';
-import { AccordionContext, AccordionDescendantContext } from './useAccordionContext';
+import { AccordionContext, AccordionDescendantContext, useAccordionContextValue } from './AccordionContext';
 
 /**
  * Function that renders the final JSX of the component
  */
 export const renderAccordion = (state: AccordionState) => {
   const { slots, slotProps } = getSlots<AccordionSlots>(state, accordionShorthandProps);
+
+  // TODO: either render function should be a hook or this hook should not be called here
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const value = useAccordionContextValue(state);
```

Comment:
you could also just construct the context value in the render function from state, it's what's being done in `Menu` and `Popover`

Reply:
That's exactly what `useAccordionContextValue` does.... same way, the problem is the same 😅

Reply:
ok, but I don't know why it is a hook